CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Change Log
 This file documents all notable changes to juttle-aws-adapter. The release numbering uses [semantic versioning](http://semver.org).
 
+## 0.3.0
+Released 2016-03-23
+
+### Minor Changes
+- Update to reflect changes in juttle 0.7.0. This release is compatible with adapter version 0.7.0. [[#12](https://github.com/juttle/juttle-aws-adapter/pull/12)].
```

Comment:
How about we keep it simple and just say "Updated to be compatible with Juttle 0.7.x".